test/edu/csus/ecs/pc2/core/execute/SandboxResultsTest.java

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 1989-2022 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.execute;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.util.Properties;
+
+import edu.csus.ecs.pc2.core.util.AbstractTestCase;
+
+/**
+ * 
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public class SandboxResultsTest extends AbstractTestCase {
+    
+    public void testLoadResults() throws Exception {
+        
+        ensureOutputDirectory();
+        String storageDirectory = getOutputDataDirectory();
+        
+//        startExplorer(storageDirectory);
+        
+        Properties props = new Properties();
+        
+        props.put(SandboxResults.COMMENT_KEY,"This is too much");
+        props.put(SandboxResults.JUDGEMENT_ACRONYM_KEY,"IO");
+        props.put(SandboxResults.RUNID_KEY,"45");
+        props.put(SandboxResults.RUNTIME_KEY,"2310");
+        
+        String propFilename = storageDirectory + File.separator + SandboxResults.SANDBOX_RESULTS_FILENAME;
+
+        FileOutputStream fileOutputStream = new FileOutputStream(propFilename, false);
+        props.store(fileOutputStream, "PC^2 Sandbox Output properties");
+        
+        assertFileExists(propFilename, "Sandbox output file");
+        
+        SandboxResults sandboxResults = new SandboxResults(new File(propFilename));
+        
+        int count = sandboxResults.getProperties().size();
+        assertEquals("Expected number of properties in "+propFilename, 4, count);
+        
+        
+//        String keyName = SandboxInput.MEMLIMIT_KEY;
+//        String str = props.getProperty(keyName);
+//        int value = Integer.parseInt(str);
+//        assertEquals("For run "+run.getNumber()+" expecting value for key "+keyName, timeLimit, value);
+
```

Comment:
Considering that this entire code base is supposed to be for supporting memory limits, it's very surprising to find the MEM_LIMIT_KEY test commented out... why?

Reply:
Prototyping PR/code to not be used.
closing PR